commons/src/main/kotlin/org/fossify/commons/extensions/Activity.kt

```diff
@@ -1675,6 +1675,9 @@ fun BaseSimpleActivity.getAlarmSounds(type: Int, callback: (ArrayList<AlarmSound
     try {
         val cursor = manager.cursor
         var curId = 1
+        val defaultAlarmSound = getDefaultAlarmSound(RingtoneManager.TYPE_ALARM)
```

Comment:
Why hardcode to `RingtoneManager.TYPE_ALARM`? The provided type should be used.

```suggestion
        val defaultAlarmSound = getDefaultAlarmSound(type)
```

Reply:
You're right, I was to focus on the Clock usecase